docs/openapi-issuer.yaml

```diff
@@ -233,6 +233,36 @@ paths:
         default:
           $ref: "#/components/responses/Unexpected"
 
+  /signature-requests/{req_id}/notification:
+    put:
+      operationId: sendNotification
+      tags:
+        - Signature Request
+      summary: Send a signature request notification to user via IO message
+      parameters:
+        - in: path
+          name: req_id
+          required: true
+          schema:
+            $ref: "#/components/schemas/Id"
+      responses:
+        "201":
+          description: The notification detail
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/NotificationDetailView"
+        "400":
```

Comment:
what if you call this endpoint twice?

Reply:
You get an error because the message has already been sent!

Reply:
with which status code? I'd use `409 confict` in case.

Reply:
It seems right.  I add `409`.